tests/metagpt/test_incremental_dev.py

```diff
@@ -0,0 +1,372 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+@Time    : 2024/01/03
+@Author  : mannaandpoem
+@File    : test_incremental_dev.py
+"""
+import os
+import subprocess
+
+import pytest
+from typer.testing import CliRunner
+
+from metagpt.const import DATA_PATH
+from metagpt.logs import logger
+from metagpt.startup import app
+
+runner = CliRunner()
+
+
+def test_refined_simple_calculator():
+    project_path = f"{DATA_PATH}/simple_add_calculator"
```

Comment:
if the files under MetaGPT/data is only for unittest, suggest move to MetaGPT/tests/data